client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -266,7 +271,7 @@ const MobileNav = () => {
               </Link>
             </div>
           )}
-          {pageName === 'home' ? (
+          {pageName === 'home' || pageName === project.name ? (
```

Comment:
This could probably look more lik `{pageName === project.name ? (` to remove the hardcoded string!